### ISSUE TITLE
Authentication error: Get accesstoken failed. Username or password error!

### ISSUE THREAD

**Issue by Scirocco3** [milo526/homebridge-tuya-web#594]
**Describe the bug**
After start is the error displayed. nothing changed in the system or username/pw
all devices are not functial. username and PW checked in the app... is ok.

**Steps To Reproduce**
after the error... this line/error displayed
Cannot acquire token, waiting 185 seconds.

**Plugin Version**
latest official
 
 
 **Homebridge Config**
{
    "name": "TuyaWebPlatform",
    "options": {
        "username": "Steffen.leicht@xxx",
        "password": "xxxxxxxxx",
        "countryCode": "49",
        "platform": "smart_life"
    },
    "defaults": [
        {
            "id": "74457336cc50e3d39579",
            "device_type": "outlet"
        },
        {
            "id": "50203372dc4f229d0f7f_7",
            "device_type": "outlet"
        },
        {
            "id": "50203372dc4f229d20ff_7",
            "device_type": "outlet"
        },
        {
            "id": "50203372dc4f229d0f7f_1",
            "device_type": "outlet"
        },
        {
            "id": "50203372dc4f229d20ff_1",
            "device_type": "outlet"
        }
    ],
    "scenes": false,
    "_bridge": {
        "username": "0E:ED:A8:42:47:38",
        "port": 33824
    },
    "platform": "TuyaWebPlatform"
}
 
 
 ```json

```
 
 **Device Config**
 
 
 **Additional Context**

**Comment by tibtiby** [comment]
having the same issue. 
[3/22/2024, 3:20:07 PM] [TuyaWebPlatform] Initializing TuyaWebPlatform platform...
[3/22/2024, 3:20:07 PM] Loading 2 accessories...
[3/22/2024, 3:20:08 PM] [TuyaWebPlatform] Cannot acquire token, waiting 185 seconds.

**Comment by imWildCat** [comment]
same issue. maybe due to their extra verification?

**Comment by doomedramen** [comment]
Same, here, let me know if there is anything I can share that would help.

**Comment by imWildCat** [comment]
Hi @doomedramen @tibtiby @Scirocco3 
Good news. I found the root cause: 
https://github.com/tuya/tuya-homebridge#:~:text=The%20TuyaSmart%20or%20SmartLife%20app%20account%20which%20registered%20using%20Google%20or%20Apple%20ID%20email%20need%20to%20link%20your%20phone%20number%20and%20use%20the%20linked%20phone%20number%20as%20username%20to%20login.

TLDR: Register another Tuya account and DO NOT use SSO login (like sign-in with Google/Apple). Just use your email to receive a code to activate the account.

Then repeat the previous steps.
